django/core/management/commands/shell.py

```diff
@@ -139,9 +140,20 @@ def get_auto_imports(self):
         ]
 
         """
+        try:
+            models = apps.get_models()
+        except AppRegistryNotReady:
+            if self.verbosity > 0:
+                self.stdout.write(
+                    "Models are not imported as not yet loaded. "
```

Comment:
Thank you for this change!

I'm going to suggest different wording here in a bit. I need to do a few things this morning, but I should have something posted in the next few hours. I'm leaving this comment for awareness.